app/src/main-process/main.ts

```diff
@@ -563,6 +563,12 @@ app.on('web-contents-created', (event, contents) => {
     event.preventDefault()
     log.warn(`Prevented new window to: ${url}`)
   })
+  // prevent link navigation within our windows
+  // see https://www.electronjs.org/docs/tutorial/security#12-disable-or-limit-navigation
+  contents.on('will-navigate', (event, url) => {
+    event.preventDefault()
+    log.warn(`Prevented navigation to: ${url}`)
```

Comment:
Is this automatically uploaded to sentry or somewhere? Would be good to know when this happens to be able to catch attackers doing nasty things...

Reply:
oo that's a good point! it will not be logged to sentry. i will add that!

Reply:
done in d397ef7!